### ISSUE TITLE
FR: Add option to only make a field editable by an admin

### ISSUE THREAD

**Issue by dploeger** [ColdTrick/profile_manager#108]
Currently there is a gap in the functionality:

* User can edit field = false => Nobody can edit the field
* Admin only field = true => Only Admins can view the field

What's missing is:
User can view field, only Admins can edit field

**Comment by jdalsem** [comment]
Good point, but i never had the usecase (yet) where i needed this combo.
